### ISSUE TITLE
Foldable 'Box'

### ISSUE THREAD

**Issue by suchwerk** [skjolber/3d-bin-container-packing#337]
How could i find optimal package for  the textiles which are foldable?

**Comment by skjolber** [comment]
Hard to say, can you translate them into a box?

**Comment by suchwerk** [comment]
Yes, sure - but its like an other option for packing like 3d-rotation. only "esay" folding like half-width but double height.

**Comment by suchwerk** [comment]
@skjolber any hint's how to implement this? I would contribute :)

**Comment by skjolber** [comment]
Well the new 2.x branch will feature predetermined rotations. I guess those could be used for different foldings too.

**Comment by skjolber** [comment]
Can you try with the new API?

**Comment by suchwerk** [comment]
Oh, watched your efforts the last month(s). I'll give it a shot the next days.

**Comment by skjolber** [comment]
@suchwerk were you able to get any further?

**Comment by suchwerk** [comment]
@skjolber not yet, too much work at the moment. but its on my schedule. hopefully in the middle of march i will give it a try.

**Comment by skjolber** [comment]
@suchwerk and now?
